### ISSUE TITLE
[Feature request] Add network support to OsmAnd protocol

### ISSUE THREAD

**Issue by Abyss777** [tananaev/traccar#2711]
There are few possible usecases:
- Test purposes
- "Self-made" devices
- Forwarding data from unsupported devices to traccar
- Possible using in Traccar Client

**Comment by tananaev** [comment]
I guess the biggest question here is how to store. Maybe we should allow serialized position in JSON body? I guess we can even create separate protocol for it.

**Comment by Abyss777** [comment]
Don't you like pass "mcc","mnc","lac","cid" as GET parameters? Or I'm missing something?

**Comment by tananaev** [comment]
There can be multiple cell towers. Also there can be WiFi access points. Now Traccar can use all that information for better accuracy.

**Comment by Abyss777** [comment]
You are right, too many parameters. May be POST with standardized object will be solution (looks like new protocol).

**Comment by tananaev** [comment]
Sounds good to me.

**Comment by SebastEnn** [comment]
OsmAnd support seems to be added. Issue can be closed.
